### PR TITLE
Add Equations/References in docstring : Sprint 22

### PR DESCRIPTION
**Pull request by athoynilimanew**
## Linked Issues

Addresses #1043 

## Description

As of the latest commit, added docstring to few functions in

- [x] `bluemira.geometry.coordinates.py`
- [x] `bluemira.materials.mixtures.py`
- [x] `bluemira/radiation_transport/radiation_tools.py`
- [x] `bluemira/coils/_tools.py`
- [x] `bluemira/coils/_field.py`

## Checklist

I confirm that I have completed the following checks:

- [x] Tests run locally and pass `pytest tests --reactor`
- [x] Code quality checks run locally and pass `pre-commit run --from-ref develop --to-ref HEAD`
- [x] Documentation built locally and checked `sphinx-build -W documentation/source documentation/build`

### CODE REVIEW

File: bluemira/radiation_transport/radiation_tools.py [hunk 1]
```diff
@@ -110,6 +109,11 @@ def upstream_temperature(
     -------
     t_upstream_kev:
         upstream temperature. Unit [keV]
+
+    Notes
+    -----
+    Reference:
+    Pitcher et al., Plasma Physics and Controlled Fusion 39.6 (1997): 779
```
Comment (je-cook) [inline bluemira/radiation_transport/radiation_tools.py:116]: Could we use the doi directive or role eg:

directive:
https://github.com/Fusion-Power-Plant-Framework/bluemira/blob/9c4ebf72f13469cd174a51d17bfc5a7bd45bdab4/bluemira/plasma_physics/rules_of_thumb.py#L84-L85

or role

https://github.com/Fusion-Power-Plant-Framework/bluemira/blob/9c4ebf72f13469cd174a51d17bfc5a7bd45bdab4/bluemira/plasma_physics/scaling_laws.py#L168

File: bluemira/radiation_transport/radiation_tools.py [hunk 2]
```diff
@@ -188,6 +190,13 @@ def target_temperature(
     -------
     t_tar:
         target temperature. Unit [eV]
+
+    Notes
+    -----
+    Reference:
+    Stangeby, "The Plasma Boundary of Magnetic Fusion
```
Comment (je-cook) [inline bluemira/radiation_transport/radiation_tools.py:197]: And here please

**Review by je-cook — CHANGES_REQUESTED** [review]
If you cant find a doi you can remove "Reference" and just directly put the reference

File: bluemira/radiation_transport/radiation_tools.py (hunk 1 above)
Comment (athoynilimanew) [reply bluemira/radiation_transport/radiation_tools.py]: done.

File: bluemira/radiation_transport/radiation_tools.py (hunk 2 above)
Comment (athoynilimanew) [reply bluemira/radiation_transport/radiation_tools.py]: done

**Review by je-cook — APPROVED** [review]

**Pull request merged**